### ISSUE TITLE
A few important enchantments needed

### ISSUE THREAD

**Issue by m0ck69** [nilaoda/N_m3u8DL-CLI#380]
Hi.
Your app is brilliant and thanks so much for developing it. However, it's missing some quite important features.
- choosing desired video quality and audio track 
- automated merging (preferably into mp4 or mov) video with audio and subtitles after successful download (then deleting files after the merge)
- downloading only audio track without video

**Comment by drjordi** [comment]
hi, you can download only audio without video. clic on audio only and then the program will download only audio.

**Comment by m0ck69** [comment]
@drjordi Nope, it doesn't work. If I choose "audio only" the program still downloads video, audio and subs. Tested with m3u8 and mpd files. Example:
`https://cloud3-atres-st.atresmedia.com/geo/assets14/2019/12/12/5D3B09E6-60AC-453B-93EF-936B8C483F43/packaged/cmaf/playlist.m3u8`

**Comment by xhlove** [comment]
I think it is encrypted media, you can only download full data then binary concat.

**Comment by m0ck69** [comment]
I don't think it's encrypted. It can be esily download by IDM or many browsers' video extensions.

**Comment by nilaoda** [comment]
The `audio only` option only takes effect when combined with ffmpeg. (Actually remove the video stream from the origin stream)

If you only want to download the **external audio stream**, you can wait for the future version...

**Comment by m0ck69** [comment]
I'm waiting. This app is so cool.

**Comment by m0ck69** [comment]
OK, I will be waiting for the future updates.
